features/tabelwaarden.feature

```diff
@@ -0,0 +1,90 @@
+Functionaliteit: Als developer wil ik waardetabellen gebruiken in plaats van enumeraties
+  zodat ik mijn code niet hoef aan te passen als een enumeratielijst wordt aangepast die ik in mijn functionaliteit niet gebruik.
+
+  Wanneer in de registratie specifieke waarden in een enumeratie voorkomen, worden deze waarden in de API doorgegeven middels een waardetabel.
```

Comment:
ik zou hier niet het woord "enumeratie" gebruiken, dat is een technische term. In de registratie (BRP/GBA-V) hebben we dat niet. 
Ook is de term "waardetabel" verwarrend, want er wordt geen tabel geleverd, alleen een combinatie van code en omschrijving.

Voorstel: "Wanneer een gegeven in de registratie een gecodeerde waarde met een beperkte set mogelijke waarden heeft, wordt dit in de API geleverd als zowel een code als de bijbehorende betekenis."

Reply:
Op zich vind ik het voorstel van Frank niks mis mee  ("beperkte set mogelijke waarden  =  enumeratie") maar ik begrijp niet waarom het woord enumeratie niet gebruikt zou mogen worden. De features bevatten toch documentatie voor developers ?  Of hebben de feature-bestanden nu ook een andere rol ? 

Verder is de essentie van wat we hier doen het vervangen van de enumeraties die nu gebruikt worden in de OAS. 
De vraag is nu hoe we dat gaan doen. 
Wat we deden : een enumeratie-waarde die zo zelfbeschrijvend mogelijk was en indien nodig in de description van de enumeratie de omschrijving bij elke enumeratiewaarde toevoegen. 

Wat we gaan doen : 
- Zowel de code als de omschrijving uit de waardetabel meeleveren. 
   - De vraag is dan of de code nog steeds zo zelfbeschrijvend mogelijk moet zijn of dat we de (betekenisloze) codes uit het  LO GBA gaan gebruiken. Ik kan me namelijk voorstellen dat deze codes ook in de vakapplicaties die deze bevragingen doen worden gebruikt en dus relevant zijn. 
   - Betekent dit dan dat de provider voor elke bevraging waarin een dergelijke code in de response voorkomt de BRP-tabellen-bevragen API meermaals moet raadplegen ?  --> en als er dan meerdere codes en een collectie van meerdere personen voorkomt kan dat veel extra API-calls opleveren. 
   - Vraag is ook of het dan voor de consumer relevant is welke mogelijk waarden die kan verwachten ? Zo nee, waarom maken we dan überhaupt gebruik van die tabellen in de BRP-bevragen API. 

of 

- Alleen de code meeleveren 
  - Consumer kan dan, indien relevant (bv voor het tonen aan de gebruiker) de omschrijving ophalen bij de BRP-tabellen-bevragen API. 

Ik denk dat het best wel zin heeft om hier toch nog een nadere ontwerp-sessie aan te wijden. Kern van mijn vraag is of dit nu 100% provider functionaliteit is of dat de consumer ook iets met die waardetabellen gaat doen.